### ISSUE TITLE
TemplateNotFound: nbconvert/templates/lab/index.html.j2

### ISSUE THREAD

**Issue by fsondej** [voila-dashboards/voila#842]
Running simply:
```
voila tiles.ipynb
```
throws:
```
ERROR:tornado.application:Uncaught exception GET / (127.0.0.1)
HTTPServerRequest(protocol='http', host='localhost:8866', method='GET', uri='/', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/home/filip/.local/lib/python3.9/site-packages/tornado/web.py", line 1704, in _execute
    result = await result
  File "/home/filip/.local/lib/python3.9/site-packages/voila/handler.py", line 154, in get
    async for html_snippet, resources in self.exporter.generate_from_notebook_node(notebook, resources=resources, extra_context=extra_context):
  File "/home/filip/.local/lib/python3.9/site-packages/voila/exporter.py", line 100, in generate_from_notebook_node
    async for output in self.template.generate_async(nb=nb_copy, resources=resources, **extra_context, static_url=self.static_url):
  File "/home/filip/.local/lib/python3.9/site-packages/jinja2/asyncsupport.py", line 35, in generate_async
    yield self.environment.handle_exception()
  File "/home/filip/.local/lib/python3.9/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/filip/.local/lib/python3.9/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/filip/.local/share/jupyter/voila/templates/lab/index.html.j2", line 1, in top-level template code
    {%- extends 'nbconvert/templates/lab/index.html.j2' -%}
jinja2.exceptions.TemplateNotFound: nbconvert/templates/lab/index.html.j2
ERROR:tornado.access:500 GET / (127.0.0.1) 250.77ms
WARNING:tornado.general:403 GET /voila/files/favicon.ico (127.0.0.1): File not whitelisted
WARNING:tornado.access:403 GET /voila/files/favicon.ico (127.0.0.1) 1.83ms
```
Link to the notebook I used:
https://github.com/fsondej/tile-away/blob/main/tiles.ipynb
It can be reproduced in this env:
https://github.com/fsondej/tile-away/blob/main/requirements.txt

**Comment by fsondej** [comment]
OK, I solved it by reinstalling nbconvert.

**Comment by shaunvxc** [comment]
ran into the same issue, what version of nbconvert did you need to install @fsondej ?

Thanks!

**Comment by SylvainCorlay** [comment]
Are you folks using the arch linux package for nbconvert?

**Comment by fsondej** [comment]
> ran into the same issue, what version of nbconvert did you need to install @fsondej ?

Now, I have `nbconvert==6.0.7`

> Are you folks using the arch linux package for nbconvert?

I installed it with pip.

**Comment by csaund** [comment]
Hey, I'm still having this issue with `nbconvert==6.0.7` and `voila==0.2.10` (and `jupyterlab==3.0.14` and `ipython==7.21.0` if those are relevant). Any other tips for resolving this? 

The exact error I'm trying to resolve is `jinja2.exceptions.TemplateNotFound: nbconvert/templates/lab/index.html.j2`. 
I've dug into the package data and found indeed, `nbconvert/templates` only contains the README and `skeleton/`, but most flavors of this issue I've found online suggest upgrading nbconvert to solve the problem. One issue I saw but can no longer find suggested copying the `html2` template directly into the top-level folder, but I can't even seem to find that template. 

Any suggestions?
